book.rb

```diff
@@ -0,0 +1,8 @@
+class Book
```

Comment:
- [ ] You should have a method for adding rentals and the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association. , it may be something like this 👇 

```ruby
def add_rental(person, date)
    Rental.new(date, self, person)
```